shell.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs ? import <nixpkgs> {} }:
+	let
+	overrides = (builtins.fromTOML (builtins.readFile ./rust-toolchain.toml));
+	libPath = with pkgs; lib.makeLibraryPath [
+		# load external libraries that you need in your rust project here
+	];
+in
+pkgs.mkShell rec {
+	# Kanidm dependencies
+	buildInputs = with pkgs; [
+		pkg-config
+		
+		git
```

Comment:
Why replace the user's git?

Reply:
If it's not used for building, then I would remove it here. If someone downloaded a tarball release and doesn't already have git preinstalled, they won't need it anyway. I thought it might at least be used for getting the version from a git tag or something.

Reply:
The shell uses the same version as the system git as long as its resolved to the same version. Removing it in any case